getting-started/merchants.md

```diff
@@ -35,4 +35,4 @@ attribution: "<!-- Icon is based on work by Freepik (http://www.freepik.com) and
 
 </div>
 
-All of the merchants are organised alphabetically. If there are any merchants that no longer provide Monero services, or a merchant that wishes to be added, please [open an issue on this website's Github repository](https://github.com/monero-project/monero-site/issues) to alert us.
+All of the merchants are organised alphabetically. If there are any merchants that no longer provide Monero services, or a merchant that wishes to be added, please [open an issue on this website's Github repository](https://github.com/monero-project/monero-site/issues) to alert us. Please, We can't accept illegal, not-trusted or scammed e-shop.
```

Comment:
I don't think this last line should be added as it has multiple issues.

1. "W" in "We" should not be capitalized.
2. Beginning with "Please," is awkward.
3. We may not want to support illegal shops, however is that really a problem here?  Remember what is illegal in one place could be legal somewhere else as well.
4. "not-trusted" is vague -- by who?
5. "scammed e-shop" means a **shop** that was scammed by somebody else.